mlflow/ml-package-versions.yml

```diff
@@ -128,6 +128,7 @@ xgboost:
     maximum: "1.7.5"
     requirements:
       ">= 0.0.0": ["scikit-learn"]
+      "< 1.6": ["pandas<2"]
```

Comment:
We need to fix a few more flavors, but this PR just fixes ones that can be fixed easily by simply adding `pandas<2`.